doc/installing.rst

```diff
@@ -24,84 +24,93 @@ KWallet, so that the sync client can login automatically.
-
-.. note:: When you upgrade from 1.7, restart Windows to ensure that all new 
+The 2.0 release of the ownCloud desktop sync client has many new features and 
+improvements.
+
+  * Multi-account support
+  * Many UI improvements
+  * Accessibility improvements (high contrast schemes)
+  * Automatic Bandwidth Throttling
+  * Don't show redundant directory entries in activity log
+  * Remove deleted accounts properly from toolbar
+  * File manager integration: Show hidden files as ignored
+  * Ignoring hidden files: Fix glitches when migrating from 1.8
+  * Show wizard when last account was deleted
+  * Do not sync down new big folders from server without user's consent
+  * Integrate Selective Sync into the default UI
+  * More reliable reconnect after timeout
+  * Use SI units for the file sizes
+  * Improve progress reporting during sync
+  * Sharing: Do not allow sharing the root folder
+  * Sharing: Show thumbnail
+  * Client Updater: Check for updates periodically, not only once per run
+  * Quota: Only refresh from server when UI is shown
+  * SSL Button: Show more information
+  * System proxy: Ask user for credentials if needed
+  * Several fixes and performance improvements in the sync engine
+  * OS X: Show file name in UI if file has invalid UTF-8 in file name 
+  * OS X: Support native finder integration for 10.10 Yosemite  
+  * Network: Try to use SSL session tickets/identifiers
+  * Windows: Support paths >255 characters
+  * Windows, OS X: Allow to not sync hidden files
+  * Windows: Remove misleading option to remove sync data
+  * Windows: Do not provoke Active Directory account locking if password changes
+  * Windows: Fix installer when installing unprivileged
+
+.. note:: When you upgrade from 1.8, restart Windows to ensure that all new 
    features are visible.
 
-Installation
-------------
+Installation Wizard
+-------------------
 
 The installation wizard takes you step-by-step through configuration options and 
-account setup. These examples are on Windows 7; note the new "Integration for 
-Windows Explorer Option". This allows Windows users to share files directly 
-from their local ownCloud folder in Windows Explorer, rather than having to 
-open a Web browser and share from the ownCloud Web interface. Mac users get the 
-same option for Finder, and Linux users for Nautilus. 
+account setup. First you need to enter the URL of your ownCloud server.
 
 .. image:: images/client1.png
-
-The next screen selects the installation folder for the client. The default is 
-fine; don't change this without a good reason.
+   :alt: form for entering ownCloud server URL
+   
+Enter your ownCloud login on the next screen.
 
 .. image:: images/client2.png
+   :alt: form for entering your ownCloud login
 
-In the next screen, enter your ownCloud server URL.
+On the Local Folder Option screen you have multiple sync options. You may sync 
+all of your files on the ownCloud server or select which files to sync. The 
```

Comment:
Actually you can only select which folders to sync. Maybe "or select which folders to sync."?
But then the sentence first talks about "all of your files" and then switches to mentioning folders. It could say:
"You may sync all of your folders and files on the ownCloud server or select which folders to sync."